package-lock.json

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
Add this file to the `.gitignore` and remove it from this PR as it isn't needed in the repository
You should also add `node_modules` to the `.gitignore` file as well